strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/MavenRepositoryManagementServiceImplTest.java

```diff
@@ -56,10 +58,10 @@ public static void cleanUp()
     public static Set<MutableRepository> getRepositoriesToClean()
     {
         Set<MutableRepository> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES_1));
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES_2));
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES_MERGE_1));
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES_MERGE_2));
+        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES_1, Maven2LayoutProvider.ALIAS));
```

Comment:
For every layout provider there's a `RepositoryFactory` implementation. (Here it would be `MavenRepositoryFactory`).
I think it would be better to use it instead...?

Reply:
maybe, but I was just fixed legacy code here and nothing more

Reply:
if you think it need to be improved, maybe OSS guys can help with this?